style.css

```diff
@@ -169,3 +188,783 @@ ul {
+  color: #1c1a19;
+  flex: none;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.line {
+  position: static;
+  width: 327px;
+  top: 107px;
+  border: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
+  width: 375px;
+  height: 3028px;
+  top: 307px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: 80px;
+  margin-bottom: 80px;
+}
+
+.intimage {
+  position: static;
+  width: 327px;
+  height: 237px;
+  left: 24px;
+  background: url('image_processing20200716-21735-f07s4k.png');
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.postopic {
+  position: static;
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.multigraph {
+  position: static;
+  width: 327px;
+  height: 147px;
+  top: 48px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: lighter;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: static;
+  width: 297px;
+  height: 40px;
+  top: 207px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  flex-grow: 0;
+  margin-right: 12px;
+}
+
+.csstag {
+  width: 49px;
+  order: 0;
+  margin-left: -12px;
+}
+
+.htag {
+  width: 56px;
+  left: 61px;
+  order: 1;
+}
+
+.bootag {
+  width: 95px;
+  left: 129px;
+  order: 2;
+}
+
+.rubytag {
+  width: 61px;
+  left: 236px;
+  order: 3;
+}
+
+.probtn {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  padding: 12px;
+  width: 130px;
+  height: 48px;
+  background: #ff6b00;
+  position: absolute;
+  left: 12px;
+  top: 560px;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  margin-left: 13px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  cursor: pointer;
+}
+
+.probtn:hover {
+  background: black;
+}
+
+.project-cards {
+  display: grid;
+  height: 2900px;
+  width: 355px;
+  top: 592px;
+  order: 1;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(6, 1fr);
+  background: url('images/backgroundimg.png');
+  background-size: 100%;
+  margin-left: 24px;
+  margin-top: 80px;
+}
```

Comment:
- [ ] [Required] Use the CSS `display: flex;` for all the projects and `display: grid;` for the parent container housing the works section.